src/Console/Component/Terminal.tsx

```diff
@@ -0,0 +1,69 @@
+import { useTerminal } from '@/utils';
+import style from './Terminal.module.css';
+import { useRef } from 'react';
+
+type TerminalValue = {
+  key: string;
+  value: string;
+};
+
+type UseTerminalReturn = {
+  currentInput: string;
+  CurrentPath: string;
+  handleChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleKeydown: (event: React.KeyboardEvent<HTMLInputElement>) => void;
+  terminalValues: TerminalValue[];
+};
+
+type TerminalProps = {
+  path: string;
+};
+
+const Terminal = ({ path }: TerminalProps) => {
+  const terminalInputRef = useRef<HTMLInputElement | null>(null);
+  const terminalRef = useRef<HTMLInputElement | null>(null);
+
+  const {
+    currentInput,
+    CurrentPath,
+    handleChange,
+    handleKeydown,
+    terminalValues,
+  }: UseTerminalReturn = useTerminal(path, terminalInputRef.current, terminalRef.current);
```

Comment:
@Shubhamabd2002 not required, define return type in the definition of useTerminal hook